src/stratis_cli/_error_reporting.py

```diff
@@ -145,7 +145,13 @@ def interpret_errors(errors):
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NameHasNoOwner"
         ):
             return "Most likely stratis is unable to connect to the stratisd D-Bus service."
-
+        if (
+            # pylint: disable=bad-continuation
+            isinstance(error, dbus.exceptions.DBusException)
+            and error.get_dbus_name() == "org.freedesktop.DBus.Error.NoReply"
+        ):  # pragma: no cover
+            return """Stratis attempted communication with the daemon, stratisd,
```

Comment:
Please use the approach for multi-line strings that is used at line 120 in this file.